src/main.rs

```diff
@@ -216,6 +223,52 @@ fn consume(inv: &mut Inventory, item_id: &str, stats: &mut Stats) {
     }
 }
 
+fn craft_item(inv: &mut Inventory, recipe_id: &str) {
+    let recipes = recipes();
+    let recipe = recipes.iter().find(|&recipe| recipe.id == recipe_id);
+
+    match recipe {
```

Comment:
Instead of using `match` you can also use the `if let` statement like this:
```rust
if let Some(recipe) = recipe {
  // ...
} else {
  // ...
}
```

Reply:
Now I like to `match` everything!!!!1!11!
but, noted.

Reply:
Now I like to match everything!!!!1!11!
but, noted.